tutorials/telegraf-output-plugin.md

```diff
@@ -0,0 +1,314 @@
+
+* Linux amd64: [deb][deb-build] [rpm][rpm-build] [binary][linux-bin-build]
+* Windows amd64 [binary/exe][windows-build]
+* MacOS amd64 [binary][mac-build]
+
+We can also provide you with a builds for:
+
+* Windows i386
+* Linux (i386, armhf, armel, arm64, static_amd64, s390x, mipsel
+* FreeBSD (amd64, i386)
+
+if you contact us using our [Public Support Slack][public-slack]
+
+Once you download the binary and extract it to a suitable location (or install the packages) we can test out the build. 
+You may have to make the file executable by running `chmod +x telegraf`.
+We can check the version of the installed Telegraf with
+
+```
+$ telegraf --version
+```
+
+If the installation was good it should print out `Telegraf 1.10.4-with-pg`.
+
+## Telegraf Configuration [](telegraf-configuration)
+
+When Telegraf is started, a config file needs to be specified. 
+The config file contains the setup for the:
+* Telegraf agent
+  * Collection interval
+  * Jitter
+  * Buffer and batch size and so on
+* Global tags added to all collected metrics from all inputs
+* Enabled outputs, processors, aggregators, inputs (and their respective configuration)
+
+A sample config file with PostgreSQL included as a plugin can be generated by executing
+
+```
+$ telegraf --input-filter=cpu --output-filter=postgresql config > telegraf.conf
+```
+
+The above command generates a config file that enables the CPU input plugin (which samples various metrics about CPU usage) and the PostgreSQL output plugin.  
+The config file also includes all available input, output, processor, and aggregator plugins, **but commented out**. So, it's easy to see how a plugin should be configured. 
+
+### Testing out the config file
+
+The selected input plugin and agent settings can be tested so they output 
+a single collection to STDOUT. By running 
+
+```
+$ telegraf --config telegraf.conf --test
+```
+
+we select the generated config file that enables only the CPU input plugin.
+And the output should look something like: 
+
+```
+> cpu,cpu=cpu0,host=local usage_guest=0,usage_idle=78.431372,usage_iowait=0,usage_irq=0,usage_softirq=0,usage_steal=0,usage_system=11.764705,usage_user=9.803921 1558613882000000000
+> cpu,cpu=cpu1,host=local usage_guest=0,usage_idle=92.156862,usage_iowait=0,usage_irq=0,usage_softirq=0,usage_steal=0,usage_system=3.921568,usage_user=3.921568 1558613882000000000
+> cpu,cpu=cpu-total,host=local usage_guest=0,usage_idle=87.623762,usage_iowait=0,usage_irq=0,usage_softirq=0,usage_steal=0,usage_system=6.435643,usage_user=5.940594 1558613882000000000
+```
+
+A line is outputted for each core of the CPU and the total. Values are presented in `key=value` pairs with the timestamp last in the row. 
+Writing to STDOUT doesn't distinguish between *tags*, which are indexed fields (cpu, host) and value *fields* (usage_quest, usage_user ...). 
```

Comment:
you can distinguish tags from fields in that output, there is a space between tags and fields